### PR TITLE
chore(target_chains/ethereum): add new networks

### PR DESCRIPTION
**Pull request by ali-bahjati**

### CODE REVIEW

File: governance/xc_admin/packages/xc_admin_common/src/chains.ts [hunk 1]
```diff
@@ -96,6 +96,9 @@ export const RECEIVER_CHAINS = {
   fuel_mainnet: 60067, // Note: Currently deployed at 50084 (fuel_testnet) but we should use 60067 for future deployments
   hemi_mainnet: 60068,
   ton_mainnet: 60069,
+  horse_dream: 60070,
+  mathematician_in_heart: 60071,
+
```
Comment (aditya520) [inline governance/xc_admin/packages/xc_admin_common/src/chains.ts:101]: Was this intentional?

File: target_chains/ethereum/contracts/deploy/zkSyncDeployEntropy.ts [hunk 2]
```diff
@@ -90,7 +90,8 @@ export default async function (hre: HardhatRuntimeEnvironment) {
   console.log("Entropy contract address:", entropyContractAddress);
 
   console.log("Saving the contract in the store...");
-  const contract = new EvmEntropyContract(chainName, entropyContractAddress);
+  const chain = DefaultStore.chains[chainName];
+  const contract = new EvmEntropyContract(chain, entropyContractAddress);
```
Comment (aditya520) [inline target_chains/ethereum/contracts/deploy/zkSyncDeployEntropy.ts:94]: Thank you for adding this. I think we should add wormhole contracts as well to the store.

**Review by aditya520 — APPROVED** [review]

File: governance/xc_admin/packages/xc_admin_common/src/chains.ts (hunk 1 above)
Comment (ali-bahjati) [reply governance/xc_admin/packages/xc_admin_common/src/chains.ts:101]: yes!

File: target_chains/ethereum/contracts/deploy/zkSyncDeployEntropy.ts (hunk 2 above)
Comment (ali-bahjati) [reply target_chains/ethereum/contracts/deploy/zkSyncDeployEntropy.ts:94]: I was too lazy to do it this time. I'll do it next time 😅

**Pull request merged**